pkg/imagestream/imagestreamhasblob.go

```diff
@@ -37,6 +38,27 @@ func (is *imageStream) HasBlob(ctx context.Context, dgst digest.Digest, requireM
 		return found
 	}
 
+	if layers, err := is.imageStreamGetter.layers(); err == nil {
```

Comment:
Why do we need to fallback to the old code?

Reply:
For 1 release we should tolerate the cluster not having /layers or the admin not configuring permissions.  We can then remove the old code path in 3.12.